mappings/net/minecraft/server/player/PlayerManager.mapping

```diff
@@ -67,7 +67,7 @@ CLASS net/minecraft/class_166 net/minecraft/server/player/PlayerManager
 	METHOD method_573 pardonIp (Ljava/lang/String;)V
 		ARG 1 playerName
 	METHOD method_574 getWhitelist ()Ljava/util/Set;
-	METHOD method_575 updateDimension (Lnet/minecraft/class_69;)V
+	METHOD method_575 remove (Lnet/minecraft/class_69;)V
```

Comment:
Why remove rather than disconnect? Please give a 50 word essay and a screenshot of the code
To support your answer

Reply:
because thsi  method removes the player from the chunk tracker, removes the player from the server and removes the player from the listof players

Reply:
Oh wait it’s removing it from all the dimensions and from the manager rather than disconnecting the actual player, isn’t it. It’s just called in the disconnect code as part of the process

Reply:
yes